bin/cheribsdtest/arm64/cheribsdtest_md.h

```diff
@@ -44,6 +44,8 @@
 
 #define	CHERI_SEAL_VIOLATION_EXCEPTION	0
 
+#define	SI_CODE_STORELOCAL	PROT_CHERI_PERM
```

Comment:
I wonder if PROT_CHERI_STORELOCAL should be defined to PROT_CHERI_PERM or undefined on Morello. Being defined, but not being indicated seems sub-optimal.

Reply:
To be honest I'm not sure why we have a separate PROT_CHERI_STORELOCAL when we don't for other permissions faults. I'd be inclined to fold it into PROT_CHERI_PERM on all architectures.